frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabParameters.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {
+  Spinner,
+  EmptyStateVariant,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateHeader,
+} from '@patternfly/react-core';
+import { PipelineRecurringRunKFv2, PipelineRunKFv2 } from '~/concepts/pipelines/kfTypes';
+import {
+  DetailItem,
+  renderDetailItems,
+} from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/utils';
+
+type PipelineRunTabParametersProps = {
+  run: PipelineRecurringRunKFv2 | PipelineRunKFv2 | null;
+};
+
+const PipelineRunTabParameters: React.FC<PipelineRunTabParametersProps> = ({ run }) => {
+  if (!run) {
+    return (
+      <EmptyState variant={EmptyStateVariant.lg} data-id="loading-empty-state">
+        <Spinner size="xl" />
+        <EmptyStateHeader titleText="Loading" headingLevel="h4" />
+      </EmptyState>
+    );
+  }
+
+  const parameters = run.runtime_config?.parameters
+    ? Object.entries(run.runtime_config.parameters)
+    : [];
+
+  if (parameters.length === 0) {
+    return (
+      <EmptyState variant={EmptyStateVariant.lg} data-id="parameters-empty-state">
+        <EmptyStateHeader titleText="No parameters" headingLevel="h2" />
+        <EmptyStateBody>This pipeline run does not have any parameters defined.</EmptyStateBody>
+      </EmptyState>
+    );
+  }
+
+  const details: DetailItem[] = parameters.map(([key, initialValue]) => {
+    let value = initialValue;
+
+    if (typeof value === 'boolean') {
+      value = value ? 'True' : 'False';
+    }
+
+    if (typeof value === 'object') {
+      value = JSON.stringify(value);
```

Comment:
pipeline spec for the run has input definition types that can be used
```json
  inputDefinitions:
    parameters:
      datastore:
        parameterType: STRUCT
      hyperparameters:
        parameterType: STRUCT
```

These should match up the the run runtime_config parameters, if not you can fall back to just a string, but if there is a match you can use the same type of parsing we do here: https://github.com/opendatahub-io/odh-dashboard/blob/33ea6969abe77445c2e0d8985d083ed22cb79e91/frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeInputOutputTab.tsx#L56-L90

make sure to create a reusable util for this then to avoid duplicate code